The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok

WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start

waiting for server to start....2025-02-14 13:04:51.706 UTC [43] LOG:  listening on IPv6 address "::1", port 5432
2025-02-14 13:04:51.706 UTC [43] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-14 13:04:51.708 UTC [43] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:04:51.715 UTC [44] LOG:  database system was shut down at 2025-02-14 13:04:51 UTC
2025-02-14 13:04:51.718 UTC [43] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE

CREATE ROLE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 13:04:52.085 UTC [43] LOG:  received fast shutdown request
2025-02-14 13:04:52.086 UTC [43] LOG:  aborting any active transactions
2025-02-14 13:04:52.087 UTC [43] LOG:  worker process: logical replication launcher (PID 50) exited with exit code 1
2025-02-14 13:04:52.088 UTC [45] LOG:  shutting down
2025-02-14 13:04:52.096 UTC [43] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 13:04:52.193 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 13:04:52.193 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 13:04:52.195 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:04:52.205 UTC [70] LOG:  database system was shut down at 2025-02-14 13:04:52 UTC
2025-02-14 13:04:52.209 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 13:04:58.283 UTC [84] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 13:04:58.283 UTC [84] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 13:04:58.718 UTC [84] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:04:58.718 UTC [84] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_c6ab76cb-1e0e-491f-a79a-e74125e65e59" WITH (FORCE)
2025-02-14 13:04:59.369 UTC [84] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:04:59.369 UTC [84] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_e6fd0f64-a2cb-4cde-b0a5-0543c96484d9" WITH (FORCE)
2025-02-14 13:04:59.793 UTC [84] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:04:59.793 UTC [84] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_ef977741-9580-4eaf-bfe1-319304c63398" WITH (FORCE)
2025-02-14 13:05:00.197 UTC [84] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:05:00.197 UTC [84] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_0cf70d16-d141-4409-b213-a4167dca92fb" WITH (FORCE)
